tests/baselines/reference/completionDetailsOfContextSensitiveParameterNoCrash.baseline

```diff
@@ -85,7 +85,7 @@
 //         return curry(getStylingByKeys, 2)(mergedStyling, ...args);
 //                                                             ^^^^
 // | ----------------------------------------------------------------------
-// | (parameter) args: any
+// | (parameter) args: []
```

Comment:
this is the correct display that we can see when hovering at this location in the playground (after the playgrounds gets typechecked first): [TS playground](https://www.typescriptlang.org/play?ts=5.5.0-dev.20240228#code/C4TwDgpgBA+jUF4oDsIDcICcDcBYAUAQJbLBYBmAhgMbQDCArppkRACYBiDy1wRA9sgCMAHgAqQgDRQASgD4oAbwJRVUABQBKAFxRGzVp268Bw8VNly8+NRuBDdEnbOsBfYqQo16TFuy48fIIATObSYsHS8koqalq6+n5GgaahEuGRlta26vaOQs6JhgEmgqIRUVaxqrkOsDDSwMGOwYW+xcZBZulZ1XZ1PU0tzjJuBB5kmFS0eu3+naYAzGFQFauLlTE2cW0G8ymCyz1rYhu92zV5qwUJc8mlyGmZp5XZcVdwjc2rrbd79100hYXuccldBt8IrskiUuuUzvI3pc6p8oEN6o1Fo5FtCOgdHis1oi+rV8l9dKjgFj1rj9g9ypliRd+hSGmjIZkqdjaQDTOULEywQMLOi1lyabpRgR3IR8CRJtMfP9YaYACyE55nMSqzbKZnxWbKhaCdXHTXhHWg97Cnkqw7ic2rS2C62s8k-W3G5BHYFa51VfXgkUcz34tJ+14kj5s9GU6mnUMPH0ZC2RwMomPfOPcv4wr1PVNW5Fu9ktTE5w15sOE-1IlnXd1i+M43N4+niWtRjPu7MYtGqxyqxNdZM-cIIgNCsmlvviykDp3D1IO8dpqezjnl2cL7VLkKEid10kbkvikHAHdD1t0wEateuhszpuny977qHrvT2Mx5uNV-X3kyhXIt60pTc0V-ftBzffl72LR9RU5SCL2gyUxllNgIGoAAbShMGgNA8KgahfBAXQ9VsFZ5HUchjF0Y8nEQBQZGkPCiFAAB+XRkAYABbAAjLAYKoqxbD6O9LBouj6whYYmNkViWE47i+MEzA3yBFMQIkkFqNonh6KDRskO5eSWKgNjlJQVShIAu1vR0j9mUcws9OkhjgzLCDsT-aCzMU9iQC46yBNsys2y6U1gUdbU4I0fTqHogA6FK8IAcwAZ10ShkBAABtABdTR5JykAAqsnjQvU5LUswTLstywrioQBRSrrMBcNoAALfhsMwzBWTcawCAgAAPMB+EwYA0XAaAAGVQGwkg0vsxAND6ABrCAQCyjQMuAFhkDSqAAB8oCobCMugM7uEw8gSHYYqzvUfbDuOs6Lqu06oFuiB7tQNhNEKyQ+hSpL8P23RuA25B+AAd2QQqCGahR+H4gArLDgGG-BMJwvDoGoQR9qgNKIGABaQCWo6ACEQAAaW23b1D6XisDJthKeptLdDRzHeBB5ktp2+jXuW77Puun7kDuh7Ae+l6DvFj7KEuqXfv+x7gdB2r6ulmH4cRgrkfkvmsZxvHcPwqBCMwKA2bq9gueW3mMfN8Z8DGiapuI4npuofDKDIZ2jrsr1lhJfjKCuoQADYxC6iA2dd-ngBR-XYYRwXbDN3hvo1uXs7UXPppumW-sLvoQ5Wr0CAUJASOYEAREj6OIDjhOk4gFOsfT6HM+QIvVBL-Py81tgh6gEey9lgHJ+r+y65Z-U+lsMmKcW5aOEwfheJptu4-oqOY-jxPk6nt3eD75ADaz1fi7ALpdpHpBFFcSfbGAM+IAAeUwaue551fu-e+qgwbpQyn0ZwC8vTyQorYNQ+FgBMGQMRUi6h17VzpozHa0hWjqAdhzau0gwYAHoABU5DSEQM0HWEBzJFjI2wEAA). If we make an edit and quickly hover over this location we can sometimes "catch" this `any` being displayed since then the quick info request happens before computing semantic diagnostics and this is exactly what this test is doing.